src/applications/burials/manifest.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  appName: '21P-530 Burials benefits form',
+  entryFile: './burials-entry.jsx',
+  entryName: 'burials',
+  receiveContentProps({ path: rootUrl }) {
+    this.rootUrl = `/${rootUrl}`;
+  }
+};
```

Comment:
Most of the changes are like this one - converting a `manifest.json` into a `manifest.js`, and then binding to the `path` property during `receiveContentProps`.